ab3d2_source/newaliencontrol.s

```diff
@@ -918,9 +913,12 @@ ACTANIMOBJ:
 				moveq	#0,d1
 				move.b	5(a3,d0.w),d1
 
-				cmp	#0,vecttimer		;animtimer decriment moved to VBlankInterrupt:
+				cmp	#0,vecttimer
+				ble.s	.zero
+				cmp	#0,vecttimer
 				bgt.s	.notzero
-
+.zero
```

Comment:
whats wrong with just bgt.s ?

Reply:
nothing, the extra check for less than was a 'just in case'.